
   HEAD
   
   
   try {
   FctA Value1 Value2 
   }
   catch {"! ERROR > A terminating error occurs." | OUT-HOST }
   
   
   try {
   FctB Value1 Value2 
   }
   catch {"! ERROR > A terminating error occurs." | OUT-HOST }
   
